core/src/test/scala/org/broadinstitute/dsde/rawls/workspace/AggregatedWorkspaceServiceSpec.scala

```diff
@@ -119,7 +126,7 @@ class AggregatedWorkspaceServiceSpec extends AnyFlatSpec with MockitoTestUtils {
     val svc = new AggregatedWorkspaceService(wsmDao)
 
     intercept[AggregateWorkspaceNotFoundException] {
-      svc.optimizedFetchAggregatedWorkspace(mcWorkspace, defaultRequestContext)
+      svc.fetchAggregatedWorkspace(mcRawlsWorkspace, defaultRequestContext)
```

Comment:
oops, in the initial PR I meant to be only testing the `optimized` method in the below stanza but missed switching these over